editor/plugins/sprite_2d_editor_plugin.cpp

```diff
@@ -706,22 +743,18 @@ bool Sprite2DEditorPlugin::handles(Object *p_object) const {
 void Sprite2DEditorPlugin::make_visible(bool p_visible) {
 	if (p_visible) {
 		sprite_editor->options->show();
+		sprite_editor->resize_region_rect->show();
```

Comment:
It's better to toggle all editor options together.
For that, you can make a single HBoxContainer for the buttons and toggle it instead.